packages/@okta/vuepress-site/docs/reference/api/policy/index.md

```diff
@@ -1161,8 +1161,43 @@ The Multifactor (MFA) Enrollment Policy controls which MFA methods are available
    }
 ```
 
+#### Policy Authenticators Settings example
+
+<ApiLifecycle access="ie" /><br>
+<ApiLifecycle access="Limited GA" /><br>
+
+> **Note:** Policy Settings are included only for those Authenticators that are enabled.
+
+```json
+   "settings": {
+     "type": "AUTHENTICATORS",
+     "authenticators": [
+       {
+         "key": "security_question",
+         "enroll": {
+           "self": "OPTIONAL"
+         }
+       },
+       {
+         "key": "okta_phone",
+         "enroll": {
+           "self": "OPTIONAL"
+         }
+       }
+     ]
+   }
+```
+
 ### Policy Settings data
 
+> **Note:** Unlike the `factors` parameter which only controls MFA methods, `authenticators` will have all authenticators available; both authentication and recovery.
```

Comment:
Maybe this note should go after the parameters table? Referring to `factors` and `authenticators` after the table will give users some context.

Also, do you want to change the note to:

```suggestion
> **Note:** The `authenticators` parameter allows you to configure all available Authenticators, including authentication, enrollment, and recovery. In contrast, the `factors` parameter only allows you to configure multifactor authentication.
```
This is just a suggestion to emphasize the `authenticators` parameter features over the `factors` one. Thanks!